### PR TITLE
chore:: update ubuntu-20.04 to ubuntu-latest

### LINKED ISSUE #991: update ci ubuntu-latest

### PR DESCRIPTION
**Pull request by fengshunli**
## Description

update runs-on:  image to `latest`

## Related issues
- Close #991

## Checklist
- [x] I've read the [guidelines for contributing](https://github.com/aquasecurity/trivy-operator/blob/main/CONTRIBUTING.md) to this repository.

### CODE REVIEW

**Comment by chen-keinan** [comment]
@fengshunli  Add a description to the issue

File: .github/workflows/build.yaml [hunk 1]
```diff
@@ -32,7 +32,7 @@ concurrency:
 jobs:
   verify-code:
     name: Verify code
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
```
Comment (chen-keinan) [inline .github/workflows/build.yaml:35]: Lets make it `ubuntu-latest`
```suggestion
    runs-on: ubuntu-latest
```

**Comment by fengshunli** [comment]
done

**Comment by chen-keinan** [comment]
> done

maybe change to `latest` in all other entries as well

**Comment by fengshunli** [comment]
The latest version is unstable, it is easy to maintain with a stable version

**Comment by chen-keinan** [comment]
> The latest version is unstable, it is easy to maintain with a stable version

should be good , we use the same in trivy and we find it working fine

**Comment by fengshunli** [comment]
done

**Review by chen-keinan — APPROVED** [review]

**Pull request merged**